base/int.jl

```diff
@@ -44,22 +44,8 @@ const BitUnsigned64T   = Union{Type{UInt8}, Type{UInt16}, Type{UInt32}, Type{UIn
 
 const BitIntegerType = Union{map(T->Type{T}, BitInteger_types)...}
 
-# >> this use of `unsigned` is defined somewhere else << the docstring should migrate there
 """
-    unsigned(T::Integer)
-
-Convert an integer bitstype to the unsigned type of the same size.
-# Examples
-```jldoctest
-julia> unsigned(Int16)
-UInt16
-julia> unsigned(UInt64)
-UInt64
-```
-""" unsigned
-
-"""
-    signed(T::Integer)
+    signed(T::Type{<:Integer})
```

Comment:
(this would align the behavior of `signed(Integer)` with `signed(Signed)`, so I think it makes reasonable enough sense)
```suggestion
signed(T::Type{<:Integer}) = Signed
```